docs/README.md

```diff
@@ -77,160 +77,24 @@ Sponsorship options are on the [Donate](DONATE.md) page. All donations are appre
 
 Have a look at [PREINSTALL-README](PREINSTALL-README.md) before proceeding.
 
-## Rebasing (Recommended)
+## Rebasing
 > [!NOTE]
 > If you don't already have a Fedora Atomic installation, use a Fedora Atomic ISO that matches your secureblue target image to install one. If you want to use a secureblue Silverblue image, start with the Fedora Silverblue ISO, Kinoite for Kinoite, Sericea (Sway Atomic) for Sericea and all the Wayblue images, and CoreOS for all the securecore images.
+> 
+> For more details on the available images, have a look at [IMAGES](IMAGES.md) before proceeding.
 
-To rebase a [Fedora Atomic](https://fedoraproject.org/atomic-desktops/) or [Fedora CoreOS](https://fedoraproject.org/coreos/) installation, follow these steps<sup>‡</sup>:
+To rebase a [Fedora Atomic](https://fedoraproject.org/atomic-desktops/) or [Fedora CoreOS](https://fedoraproject.org/coreos/) installation, download the script below:
 
-> [!IMPORTANT]
-> The **only** supported tag is `latest`.
+[![Download](https://shields.io/badge/-Download-blue?style=for-the-badge&logo=download&logoColor=white)](https://github.com/secureblue/secureblue/releases/download/4.1.0/install_secureblue.sh)
 
-- First rebase to the unsigned image, to get the proper signing keys and policies installed:
-  ```
-  rpm-ostree rebase ostree-unverified-registry:ghcr.io/secureblue/IMAGE_NAME:latest
-  ```
-- Reboot to complete the rebase:
-  ```
-  systemctl reboot
-  ```
-- Then rebase to the signed image, like so:
-  ```
-  rpm-ostree rebase ostree-image-signed:docker://ghcr.io/secureblue/IMAGE_NAME:latest
-  ```
-- Reboot again to complete the installation:
-  ```
-  systemctl reboot
-  ```
-<sup>‡</sup> Replace `IMAGE_NAME` with the *full name* of your preferred image from the [list below](README.md#images).
+Then, mark it as executable (Right click -> Properties):
```

Comment:
not needed if running `bash install_secureblue.sh` (don't need to mark executable when specifying bash).

Reply:
good point